inventory_reporting/reports/inventory_report.py

```diff
@@ -173,6 +174,16 @@ def generate_storable_report(self, workbook, wizard):
                     ["product_id.detailed_type", "=", "product"],
                 ],
             },
+            {
+                "name": _("Unbuild"),
+                "filter": [
+                    ["accounting_date", ">=", wizard.date_start],
+                    ["accounting_date", "<=", wizard.date_end],
+                    ["product_id.active", "=", True],
+                    ["product_id.detailed_type", "=", "product"],
+                    ["stock_move_id.unbuild_id", "!=", False],
```

Comment:
@rinaldifirdaus @AungKoKoLin1997 Why don't you make a common method to return the base domain (or define the base domain at the beginning of the method) to which you make adjustments depending on the report type?  We shouldn't be repeating the same conditions several times.